ingest/src/test/java/com/microsoft/azure/kusto/ingest/E2ETest.java

```diff
@@ -141,7 +141,8 @@ public static void setUp() {
     @AfterAll
     public static void tearDown() {
         try {
-            queryClient.executeToJsonResult(DB_NAME, String.format(".drop table %s ifexists skip-seal", tableName));
+            new ExponentialRetry<DataServiceException, DataClientException>(2)
```

Comment:
did it fail for you?>

Reply:
I think it's an holdover from when the tests failed, I'll undo it